tests/tracer/test_processor.py

```diff
@@ -1,55 +1,71 @@
+from typing import Any
+
 import mock
+import pytest
 
 from ddtrace import Span
 from ddtrace.filters import TraceFilter
-from ddtrace.internal.processor import TraceProcessor
+from ddtrace.internal.processor import Processor
+from ddtrace.internal.processor import TraceFilterProcessor
+from ddtrace.vendor import attr
 
 
-def test_no_filters():
```

Comment:
We actually have coverage of these removed tests with the integration test case `test_filters` in `test_tracer.py`